UPGRADING.md

```diff
@@ -1,5 +1,35 @@
 # Upgrade guidelines
 
+# 0.100.0 to 0.100.1
+
+Feature gates for controlling instrumentation in the operator have been replaced with command-line arguments.
+This update primarily affects users of non-Splunk distributions like apache-httpd, go, nginx, and python, which are now disabled by default.
+
+If you were using `operator.manager.featureGates`, convert previous feature gates to command-line arguments in `operator.manager.extraArgs`. See below for summary of release changes:
+- `+operator.autoinstrumentation.java` becomes `--enable-java-instrumentation=true`
+- `+operator.autoinstrumentation.nodejs` becomes `--enable-nodejs-instrumentation=true`
+- `+operator.autoinstrumentation.dotnet` becomes `--enable-dotnet-instrumentation=true`
+- `+operator.autoinstrumentation.apache-httpd` becomes `--enable-apache-httpd-instrumentation=false`
+- `-operator.autoinstrumentation.go` becomes `--enable-go-instrumentation=false`
+- `-operator.autoinstrumentation.nginx` becomes `--enable-nginx-instrumentation=false`
+- `+operator.autoinstrumentation.python` becomes `--enable-python-instrumentation=false`
```

Comment:
Doesn't seem like all `-/+` match the booleans

Reply:
It purposely does not based off of a product decision moving forward. Line 6 mentions the details for these mismatches, I'll see if I can clarify more.